homeassistant/components/hassmpris/const.py

```diff
@@ -0,0 +1,17 @@
+"""Constants for the MPRIS media playback remote control integration."""
+
+import logging
+
+LOGGER = logging.getLogger(__package__)
+
+DOMAIN = "hassmpris"
+ENTRY_CLIENT = "client"
+ENTRY_MANAGER = "manager"
```

Comment:
is this manager for entries or entities? both seem to be used

Reply:
I don't understand the difference?  I could use some help here.

Reply:
this is a constant that you define, so it can be anything. usually, entries relate to config entries (usually one per who system) where entities are the various lights, switches, covers, etc.

Just thought the code would be more clear if you define what it manages

Reply:
I think I got it.  Fixed.